roles/ceph-nfs/tasks/pre_requisite_non_container.yml

```diff
@@ -223,3 +223,10 @@
     owner: "root"
     group: "root"
     mode: "0755"
+
+- name: stop nfs server service
+  systemd:
+    name: nfs-server
```

Comment:
the unit name on ubuntu is `nfs-kernel-server`

Reply:
@alimaredia sorry, I think i misled you. `nfs-server` seems to be ok on Debian based os (actually, it looks like both are ok).
The issue here is that if you try to stop a service which doesn't exist with module `systemd` its going to fail.